rmk/src/via/protocol.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "vial")]
```

Comment:
It's unnecessary because this file is already behind vial feature gate in `via/mod.rs`